files/justfiles/utilities.just

```diff
@@ -146,3 +146,87 @@ toolbox-assemble CONTAINER="prompt" ACTION="create" FILE="/etc/toolbox/toolbox.i
 with-standard-malloc APP:
     #!/usr/bin/bash
     bwrap --dev-bind / / --ro-bind /dev/null /etc/ld.so.preload {{ APP }}
+
+# Add the unfiltered Flathub flatpak repo
+enable-flathub-unfiltered:
+    #!/usr/bin/bash
+    flatpak remote-add --if-not-exists --user flathub https://flathub.org/repo/flathub.flatpakrepo
+
+# Install Steam via choice of 3 methods
+install-steam:
+    #!/bin/bash
+    if [ -n "$(rpm-ostree status | grep 'kinoite\|sericea\|silverblue\|sway')" ] ; then
+        echo "Steam requires Xwayland, which your variant of secureblue has disabled by default."
+        echo "If you have not already done so, please run 'ujust toggle-xwayland' and reboot to enable it."
+        toggle_xwayland_now=""
+        read -p "Would you like to toggle Xwayland now? [y/N] " toggle_xwayland_now
```

Comment:
this should detect whether xwayland is already enabled, and skip all of this if it is